src/main/java/bank/backend/bms/controllers/UserRoleController.java

```diff
@@ -0,0 +1,76 @@
+package bank.backend.bms.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import bank.backend.bms.services.UserRoleService;
+import jakarta.validation.Valid;
+
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import bank.backend.bms.models.UserRole;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/userroles")
```

Comment:
can you make a simpler route mapping as "/api/roles"

Reply:
done